xml/System.Net.Sockets/Socket.xml

```diff
@@ -4683,16 +4683,17 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
   
  If you call the <xref:System.Net.Sockets.Socket.%23ctor%2A> constructor multiple times with the same byte array as the argument for each call, you will create multiple managed <xref:System.Net.Sockets.Socket> instances with the same underlying socket. This practice is strongly discouraged.  
   
- If the process creating the socket uses asynchronous methods (<xref:System.Net.Sockets.Socket.BeginReceive%2A> or <xref:System.Net.Sockets.Socket.BeginSend%2A>), the process must first set the <xref:System.Net.Sockets.Socket.UseOnlyOverlappedIO%2A> property to true; otherwise, the socket is bound to the completion port of the creating process, which may cause an <xref:System.ArgumentNullException> to be thrown on the target process.  
+On .NET Framework, if the process creating the socket uses asynchronous methods (<xref:System.Net.Sockets.Socket.BeginReceive%2A> or <xref:System.Net.Sockets.Socket.BeginSend%2A>), the process must first set the <xref:System.Net.Sockets.Socket.UseOnlyOverlappedIO%2A> property to `true`. Otherwise, the socket is bound to the completion port of the creating process, which may cause an <xref:System.ArgumentNullException> to be thrown on the target process. On .NET 5+, socket duplication doesn't work with [I/O completion ports](/windows/win32/fileio/i-o-completion-ports).
```

Comment:
I would remove the references to `BeginReceive` and `BeginSend`. The range of async API-s is much wider, even on Framework.

> On .NET 5+, socket duplication doesn't work with [I/O completion ports](/windows/win32/fileio/i-o-completion-ports).

This statement is not specific to .NET 5+ but rather to the Windows OS. I would replace with a text that focuses on .NET capabilities:

```suggestion
On .NET Framework, if the process creating the socket uses asynchronous methods, the process must first set the <xref:System.Net.Sockets.Socket.UseOnlyOverlappedIO%2A> property to `true`. Otherwise, the asynchronous method will bind the socket to an [I/O completion port](/windows/win32/fileio/i-o-completion-ports) port of the creating process, which may cause an <xref:System.ArgumentNullException> to be thrown on the target process.

On .NET Core 1.0-3.1 the method is unsupported.

On .NET 5+, the has limited support on Windows. Unlike on .NET Framework the <xref:System.Net.Sockets.Socket.UseOnlyOverlappedIO%2A> property is a NOP and, the process creating the socket must not call any asynchronous methods on it. A call to an asynchronous operation will always bind it to an [I/O completion port](/windows/win32/fileio/i-o-completion-ports) of the creating process, which may cause an <xref:System.ArgumentNullException> to be thrown on the target process.
```